### PR TITLE
Implement new organizations API entry point

### LINKED ISSUE #85: Is it possible to figure out which organisations are members from the API?
It would be useful to know which videos belong to member organisations.  Is there a way to do this using the API?  If not, could the API be extended?

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
Fixes #85

### CODE REVIEW

File: fkbeta/fkws/views.py [hunk 1]
```diff
@@ -345,3 +348,49 @@ class VideoFileDetail(generics.RetrieveUpdateDestroyAPIView):
...
+            if 'true' == fkmember:
+                queryset = queryset.filter(fkmember=True)
+            else:
+                queryset = queryset.filter(fkmember=False)
+        orgnr = self.request.query_params.get('orgnr')
+        if orgnr and orgnr.isdigit():
+            queryset = queryset.filter(orgnr=int(orgnr))
+        return queryset
+
+    def perform_create(self, serializer):
```
Comment (odinho) [inline fkbeta/fkws/views.py]: Remove this.

File: fkbeta/fkws/views.py [hunk 2]
```diff
@@ -345,3 +348,49 @@ class VideoFileDetail(generics.RetrieveUpdateDestroyAPIView):
...
+                  all items.  Default is 50 items.
+
+    `ordering` - Order results by specified field.  Prepend a minus for
+                 descending order.  I.e. `?ordering=-starttime`.
+    """
+    queryset = Organization.objects.all()
+    serializer_class = OrganizationSerializer
+    filter_backends = (filters.OrderingFilter,)
+    pagination_class = Pagination
+#    permission_classes = (IsInOrganizationOrReadOnly,)
```
Comment (odinho) [inline fkbeta/fkws/views.py]: Why do you not have this enabled?

File: fkbeta/fkws/tests.py [hunk 3]
```diff
@@ -331,6 +331,13 @@ def _get_upload_token_helper(self, thing_tests):
             self.assertEqual(status, r.status_code)
```
Comment (odinho) [inline fkbeta/fkws/tests.py]: A lot of these tests, that list "reverse names", should also have this new endpoint listed, i.e. `test_anonymous_cannot_add`, `test_nuug_user_can_edit_own_things`, `test_nuug_user_cannot_edit_nonowned_things`, `test_anonymous_cannot_edit` etc.

Comment (petterreinholdtsen) [reply fkbeta/fkws/tests.py]: Most of these functions work on -details views, not lists.  I did not plan to add the ability to add or update organizations using the API, are all these still needed also for lists?

**Comment by toresbe** [comment]
Hey guys, this PR seems stale so I am archiving it. If you want to revive the thread, please feel free to do so. But I believe that changes I have made later have satisfied these concerns.

**Pull request closed without merging**